Azure/azure-cli
# [SQL] `az sql db`: Add `--preferred-enclave-type` argument

**Related command**


- az sql db create
- az sql db show

**Description**

- Adds support for PreferredEnclaveType parameter for Database CRUD operations. 

**Testing Guide**

- az sql db create --preferred-enclave-type "VBS"
- az sql db show

---

This checklist is used to make sure that common guidelines for a pull request are followed.
